packages/components/src/mobile/utils/test/index.native.js

```diff
@@ -76,6 +76,7 @@ describe( 'isContainerRelated', () => {
 		'core/column',
 		'core/buttons',
 		'core/button',
+		'core/quote',
```

Comment:
@carolinan, this lists should only contain blocks that support wide alignment. You can see the relevant function the currently failing test uses to check for each of the block's alignment:

https://github.com/WordPress/gutenberg/blob/1fb1343c42a59d5062660a7da20a4bff4d06e369/packages/components/src/mobile/utils/alignments.native.js#L34-L44  

As the quote block does not support wide alignment, it should be removed from this list.

Reply:
Thank you! I understand now. This alignment was supported and then I was asked to remove it and I missed this.

Reply:
Thanks for making that change!